tests/jballerina-unit-test/src/test/resources/test-src/klass/simple_class.bal

```diff
@@ -70,6 +70,33 @@ class Per {
     int i = 0;
 }
 
+final int classI = 111222;
+
+class ModuleVariableReferencingClass {
+    int i = classI;
+}
+
+function value(int k = classI) returns int {
+    return k;
+}
+
+ModuleVariableReferencingClass c1 = new;
+
+function testModuleVariableReferencingClass() {
+    ModuleVariableReferencingClass c = new;
+    assertEquality(c.i, 111222);
```

Comment:
Need to swap values? First arg is meant to be expected and second actual?

Reply:
Fixed.